main.go

```diff
@@ -440,7 +440,13 @@ This command wraps "go %s". Below is its help:
 		return nil, err
 	}
 
-	if err := setListedPackages(args); err != nil {
+	binaryBuildID, err := buildidOf(cache.ExecPath)
+	if err != nil {
+		return nil, err
+	}
+	cache.BinaryContentID = decodeHash(splitContentID(binaryBuildID))
```

Comment:
Why was this code added here?

Reply:
Because it's unrelated to ListedPackages, and running it twice for the runtime builds won't be useful

Reply:
As in, this code is simply moved, not new :)

Reply:
Ah, I didn't realize you moved it from shared.go